test/cpp/jit/test_gpu.cpp

```diff
@@ -17521,6 +17521,48 @@ TEST(NVFuserTest, FusionFloatPow_CUDA) {
       __FILE__);
 }
 
+TEST(NVFuserTest, FusionIssue1127_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  const int numel = 4;
+
+  auto tv0 = makeConcreteTensor({numel});
+  fusion.addInput(tv0);
+
+  auto tv1 = sum(tv0, {0});
+  auto tv2 = broadcast(tv1, {true});
+
+  auto tv3 = makeConcreteTensor({numel, numel});
+  fusion.addInput(tv3);
+
+  auto tv4 = sum(tv3, {1});
+
+  auto tv5 = add(tv2, tv4);
+  fusion.addOutput(tv5);
+
+  tv1->axis(0)->parallelize(ParallelType::TIDx);
+  tv2->axis(0)->parallelize(ParallelType::TIDx);
+  tv4->axis(1)->parallelize(ParallelType::TIDx);
```

Comment:
This test case looks invalid in another aspect, which seems to be another way to look at the same story. 

`T4` should fail parallel check in this case since `tv4->axis(1)` is `TIDx`, and `tv4->axis(0)` is mappable to `tv2->axis(0)` by caParallelMap.

Reply:
This is really a weird fusion, but it doesn't have any computAt, so nothing is mapped by the parallel map, which only maps CA axes.

Reply:
I see parallel map doesn't help here. Thanks for the clarification.